ShippingApp/src/main/java/com/javaschool/common/ErrorResponse.java

```diff
@@ -0,0 +1,22 @@
+package com.javaschool.common;
+
```

Comment:
tip: Take a look to lombok library to avoid boilerplate code.
[https://projectlombok.org/](https://projectlombok.org/)

Reply:
The use of Lombok was added.